lib/service/service.go

```diff
@@ -1929,20 +1929,26 @@ func (process *TeleportProcess) initAuthService() error {
 
 	// clean up unused descriptors passed for proxy, but not used by it
 	warnOnErr(process.closeImportedDescriptors(teleport.ComponentAuth), log)
-	if cfg.Auth.EnableProxyProtocol {
-		log.Infof("Starting Auth service with external PROXY protocol support.")
+
+	if cfg.Auth.PROXYProtocolMode == multiplexer.PROXYProtocolOn {
+		log.Info("Starting Auth service with external PROXY protocol support.")
+	}
+	if cfg.Auth.PROXYProtocolMode == multiplexer.PROXYProtocolUnspecified {
+		log.Warn("'proxy_protocol' unspecified." +
```

Comment:
Update this message to explain how to turn it on or off explicitly in the config.